cime_config/runseq/runseq_CG.py

```diff
@@ -81,15 +78,18 @@ def runseq(case, coupling_times):
             outfile.write ("::                                  \n")
 
     elif comp_wav == 'ww' or comp_wav == "dwav":
+
+        if atm_cpl_dt < ocn_cpl_dt:
+            expect(False, "for C or G compset assume that atm_cpl_ct >= ocn_cpl_dt")
```

Comment:
I find this error message a bit confusing.   You say assume, but what you mean is its required.

Reply:
right - I'll fix it and push the change back

Reply:
done